### DISCUSSION TITLE
Burner curve gone

### DISCUSSION THREAD

**Discussion by Will-Scream** [artisan-roaster-scope/artisan#1169] [Q&A] [answered]
Hello, 
I have been using Artisan with a Loring S35 for about a year with no problems.

Last week I noticed that during roasts the Burner curve (green) was no longer seen.

The burner readings on the LCD's on the right side show that it is being read. But no curve line is forming.
I haven't changed the colors and settings. so why isnt it showing? The other curves are normal as usual.

Any help is appreciated :)

![Screenshot (18)](https://github.com/artisan-roaster-scope/artisan/assets/95874675/96d35d5a-c087-40ae-b4ea-3acc820dc0bd)
![Screenshot (19)](https://github.com/artisan-roaster-scope/artisan/assets/95874675/4d0c6e77-3b5f-48f7-92cd-65866eec33d3)

**Comment by bigl00z3** [comment]
Try clicking on the LCD on the right of the screen. You can hide/show curves by doing this. Maybe you clicked by mistake in the past.
![image](https://github.com/artisan-roaster-scope/artisan/assets/8886922/dc95f603-289a-408a-966d-8b9098280a44)

Nico

**Reply by Will-Scream** [reply]
Thanks, but that didn't seem to work. Maybe I'm clicking it wrong (?) 
Also, some additional info. When I End the Roast, the Burner doesn't show up on the Legend. 
![Screenshot (21)](https://github.com/artisan-roaster-scope/artisan/assets/95874675/d8d76e50-058c-4a02-aa9a-0bd40a2170c8)

**Comment by MAKOMO** [comment, marked as answer]
Hm. You seem to have played around a bit with the settings, mostly with the colors. By default the legend looks like this:

<img width="246" alt="Screenshot 2023-05-17 at 12 08 04" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/6e17ed91-7c5b-4e2a-8ce7-427f357198a8">

The "Inlet" and "Stack" entries seem to be missing in your version, but zooming in one sees that they are there, just rendered black on grey. To show them go to the Extra Devices tab (menu Config >> Device, 2nd tab) and choose reasonable values for Color 1 and Color 2 that can be seen on your black background. If you want to hide curves, either click the corresponding LCD (if visible) to toggle its visibility or untick the corresponding Curve 1 or Curve 2 entries here in this Extra Device config dialog. Here you can also activate/deactivate the corresponding LCDs by your liking.

Here is the default Extra Device setup for the Loring

<img width="1063" alt="Screenshot 2023-05-17 at 12 06 10" src="https://github.com/artisan-roaster-scope/artisan/assets/13909011/daf48d80-7667-431c-98ab-b4e5e1f693a2">

The {3} in the Label 1 of the first extra device is substituted by the name of the last event name (here "Burner").

**Reply by Will-Scream** [reply]
Thank you. That was exactly it!
Going through settings worked very well.
Clicking on the LCD had no effect though.
